mypy/modulefinder.py

```diff
@@ -944,23 +884,4 @@ def typeshed_py_version(options: Options) -> Tuple[int, int]:
     """Return Python version used for checking whether module supports typeshed."""
     # Typeshed no longer covers Python 3.x versions before 3.6, so 3.6 is
     # the earliest we can support.
-    if options.python_version[0] >= 3:
-        return max(options.python_version, (3, 6))
-    else:
-        return options.python_version
-
-
-def filter_redundant_py2_dirs(dirs: List[str]) -> List[str]:
-    """If dirs has <dir>/@python2 followed by <dir>, filter out the latter."""
-    if len(dirs) <= 1 or not any(d.endswith(PYTHON2_STUB_DIR) for d in dirs):
-        # Fast path -- nothing to do
-        return dirs
-    seen = []
-    result = []
-    for d in dirs:
-        if d.endswith(PYTHON2_STUB_DIR):
-            seen.append(os.path.dirname(d))
-            result.append(d)
-        elif d not in seen:
-            result.append(d)
-    return result
+    return max(options.python_version, (3, 6))
```

Comment:
Typeshed no longer supports 3.6 (the comment immediately above is also out of date)

```suggestion
    return max(options.python_version, (3, 7))
```

Reply:
Good point, I'll do this in another PR